### PR TITLE
updated documentation

### PR DESCRIPTION
**Pull request by kathrynkodama**
- update documentation for IntelliJ Marketplace
image

- minor adjustment to `Liberty Explorer` to only display tree view if valid config files are found in the project

Signed-off-by: Kathryn Kodama <kathryn.s.kodama@gmail.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+```bash
+git commit -s
+```
+
+### Coding Standards
+
+This project follows Eclipse standard Java language [coding conventions](https://wiki.eclipse.org/Coding_Conventions).
+
+Please note:
+ - all PRs must pass Java CheckStyle checks
```
Comment (ericglau) [inline CONTRIBUTING.md:42]: Are there actually Java CheckStyle checks in this project?

File: README.md [hunk 2]
```diff
@@ -12,11 +12,46 @@ and [Liberty Gradle](https://github.com/OpenLiberty/ci.gradle/blob/master/docs/l
...
+| ---------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Start                        | Starts dev mode.                                                                                                                                                                                                                                                                                                             |
+| Start…​                      | Starts dev mode with custom parameters. Supported parameters can be found in the documentation for the [dev goal of the Liberty Maven Plugin](https://github.com/OpenLiberty/ci.maven/blob/master/docs/dev.md#dev) and the [libertyDev task of the Liberty Gradle Plugin](https://github.com/OpenLiberty/ci.gradle/blob/master/docs/libertyDev.md#command-line-parameters). |
```
Comment (ericglau) [inline README.md:34]: If the intent is to link to the parameters sections of the docs, use `https://github.com/OpenLiberty/ci.maven/blob/master/docs/dev.md#additional-parameters`

File: src/main/resources/META-INF/plugin.xml [hunk 3]
```diff
@@ -2,11 +2,24 @@
...
     <description><![CDATA[
-    Open Liberty Tools is an IntelliJ extension that detects your Open Liberty projects and allows you to
-     interact with Liberty Dev mode via the Open Liberty Tools dashboard.<br>
+    <b>This is a preview release.</b>
+    <br> Adds support for
+    <a href="https://openliberty.io/">Open Liberty</a>
+    projects.<br>
+     The following features are available through the Liberty Dev Dashboard:
+     <ul>
+     <li> View supported <a "href=https://github.com/OpenLiberty/ci.maven/blob/master/docs/dev.md#dev"><code>liberty-maven-plugin</code></a>(version <code>3.1</code> or higher) or
```
Comment (ericglau) [inline src/main/resources/META-INF/plugin.xml:16]: For this description, I think we can remove the mention of what Liberty plugin versions it requires.

File: README.md [hunk 4]
```diff
@@ -12,11 +12,46 @@ and [Liberty Gradle](https://github.com/OpenLiberty/ci.gradle/blob/master/docs/l
 
 Note that this extension requires the [Integrated Terminal plugin](https://plugins.jetbrains.com/plugin/13123-terminal) to be enabled.
 
+## Quick Start
+
+- Search for _Open Liberty Tools_ in IntelliJ **Settings > Plugins > Marketplace** and click the Install button.
+- Open Liberty supported projects will appear in the Liberty Dev Dashboard on the side bar
+- Click a project in the Liberty Dev Dashboard to view the available commands
```
Comment (ericglau) [inline README.md:19]: Make these consistent with whether they end with periods or not.

File: src/main/resources/META-INF/plugin.xml [hunk 5]
```diff
@@ -2,11 +2,24 @@
...
+     <a "href=https://github.com/OpenLiberty/ci.gradle/blob/master/docs/libertyDev.md"><code>liberty-gradle-plugin</code></a>(version <code>3.0</code or higher) projects in the workspace
```
Comment (ericglau) [inline src/main/resources/META-INF/plugin.xml]: In your screenshot, it looks like the end of this sentence is cut off.

Comment (kathrynkodama) [reply src/main/resources/META-INF/plugin.xml]: Thank you - I was missing the closing `>`

File: src/main/resources/META-INF/plugin.xml [hunk 6]
```diff
@@ -2,11 +2,24 @@
...
     <description><![CDATA[
-    Open Liberty Tools is an IntelliJ extension that detects your Open Liberty projects and allows you to
-     interact with Liberty Dev mode via the Open Liberty Tools dashboard.<br>
+    <b>This is a preview release.</b>
+    <br> Adds support for
+    <a href="https://openliberty.io/">Open Liberty</a>
+    projects.<br>
+     The following features are available through the Liberty Dev Dashboard:
+     <ul>
+     <li> View supported <code>liberty-maven-plugin</code>(version <code>3.1</code> or higher) or
```
Comment (ericglau) [inline src/main/resources/META-INF/plugin.xml:16]: The links to `<a "href=https://github.com/OpenLiberty/ci.maven/blob/master/docs/dev.md#dev">` were fine.  I meant we can remove the parts that say `(version <code>3.1</code> or higher)`

File: src/main/resources/META-INF/plugin.xml (hunk 5 above)
Comment (kathrynkodama) [reply src/main/resources/META-INF/plugin.xml]: image

**Pull request merged**